talaiot/src/main/kotlin/com/cdsap/talaiot/publisher/timeline/ExecutionResult.kt

```diff
@@ -0,0 +1,3 @@
+package com.malinskiy.marathon.report.debug.timeline
```

Comment:
the package name is not correct

Reply:
same apply for other data classes of TimeLine

Reply:
good catch